src/main/java/org/testng/internal/PackageUtils.java

```diff
@@ -31,7 +31,7 @@
   private static String[] testClassPaths;
 
   /** The additional class loaders to find classes in. */
-  private static final List<ClassLoader> classLoaders = new Vector<>();
+  private static final List<ClassLoader> classLoaders = Lists.newArrayList();
```

Comment:
yes i was contemplating on that. But I was also feeling that perhaps we don't need a synchronization mechanism. If needed, I can try adding explicit sync locks around the places wherein data is being added into this list. pls let me know.